### ISSUE TITLE
Feature: request current svg for user

### ISSUE THREAD

**Issue by 2016rshah** [akerl/githubchart#38]
It would be really cool to be able to put your own chart on your own webpage.

Basically if we could just wrap this gem into an api that returns the svg file for the specified user, your html you could just say `<img src="http://githubchart.me/2016rshah">` and you would see the svg on the page.

Does that make sense?

**Comment by akerl** [comment]
That sounds rad, but I'm not sure I can commit to the uptime/maintenance requirements of providing this as a persistent service like that. If you're interested in setting that up, I'd be stoked and would happily link to your service in the README for this library

**Comment by 2016rshah** [comment]
This might be a simple question, but how can I use this gem from ruby code rather than from the command line?

**Comment by akerl** [comment]
Depends how fancy you want to be. The short version is:

```
require 'githubchart'
chart = GithubChart.new.svg
# chart is now an object that can be written to a file or otherwise manipulated
File.open('./output_file', 'w') { |file| file << chart }
```

If you need to pass in options (like username):

```
require 'githubchart'
chart = GithubChart.new(user: '2016rshah').svg
# chart is now an object that can be written to a file or otherwise manipulated
File.open('./output_file', 'w') { |file| file << chart }
```

There are also other options that can be passed to GithubChart.new, which can be checked out in `./bin/githubchart`

**Comment by 2016rshah** [comment]
Tah-dah! https://github.com/2016rshah/githubchart-api

This is yours (which is very impressive, by the way!):

![akerl's chart](http://ghchart.rshah.io/akerl.svg)

**Comment by akerl** [comment]
That's awesome! Dropped a link to that in the readme.

Also noticed that I messed up the month labels a bit on the very left side, I just released version 1.0.1 that should correct that.

**Comment by 2016rshah** [comment]
I pulled the latest version, thanks!
